src/main/java/ch/njol/skript/bukkitutil/PlayerUtils.java

```diff
@@ -27,32 +35,22 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import org.bukkit.Bukkit;
-import org.bukkit.GameMode;
-import org.bukkit.Material;
-import org.bukkit.entity.Player;
-import org.eclipse.jdt.annotation.Nullable;
-
-import com.google.common.collect.ImmutableList;
-
-import ch.njol.skript.Skript;
-import ch.njol.skript.util.Task;
-
 /**
  * TODO check all updates and find out which ones are not required
- * 
+ *
  * @author Peter Güttinger
  */
+@SuppressWarnings({"ThrowableNotThrown", "FinalStaticMethod"})

```

Comment:
Nonstandard warnings suppressed. And besides, final static methods are a warning... ?

Reply:
Ah, I see.